usr/share/rear/layout/save/default/335_remove_excluded_multipath_vgs.sh

```diff
@@ -19,9 +19,9 @@ while read lvmdev name mpdev junk ; do
     # Remember, multipath devices from a volume group that is "excluded" should be 'commented out'
     device=$(echo $mpdev | cut -c1-45)
     while read LINE ; do
-        # Now we need to comment all lines that contain "$devices" in the LAYOUT_FILE
+        # Now we need to comment all lines that contain "$device" in the LAYOUT_FILE
         sed -i "s|^$LINE|\#$LINE|" "$LAYOUT_FILE"
-    done < <(grep -w "$device" $LAYOUT_FILE | grep -v "^#")
+    done < <(grep "\s$device\s" $LAYOUT_FILE | grep -v "^#")
```

Comment:
I wonder if `\s` is supported by all grep versions
because "man grep" does not mention if for GNU grep 3.1
that I have on openSUSE Leap 15.3

In contrast to the GNU Grep 3.7 documentation lists it in
https://www.gnu.org/software/grep/manual/html_node/The-Backslash-Character-and-Special-Expressions.html

But it seems to work for me with GNU grep 3.1
```
# for v in beginfoo foo-bar foo_baz  foo1.2  fooend ; do echo "beginfoo foo-bar foo_baz foo1.2 fooend" | echo "'$( grep -o "\s$v\s" )'" ; done
''
' foo-bar '
' foo_baz '
' foo1.2 '
''
```

Reply:
GNU grep 2.5.1 that SUSE has in SLES 10 SP4 does not support `\s`
```
# echo " foo " | grep "\sfoo\s" && echo Y || echo N
N
```

GNU grep 2.7 that SUSE has in SLES 11 SP4 does support `\s`
```
# echo " foo " | grep "\sfoo\s" && echo Y || echo N
 foo 
Y

# for v in beginfoo foo-bar foo_baz foo1.2 fooend ; do echo "beginfoo foo-bar foo_baz  foo1.2  fooend" | echo "'$( grep -o "\s$v\s" )'" ; done
''
' foo-bar '
' foo_baz '
' foo1.2 '
''
```

GNU grep 2.16 that SUSE has in SLES 12 SP5 does support `\s`
```
# echo " foo " | grep "\sfoo\s" && echo Y || echo N
 foo 
Y

# for v in beginfoo foo-bar foo_baz foo1.2 fooend ; do echo "beginfoo foo-bar foo_baz  foo1.2  fooend" | echo "'$( grep -o "\s$v\s" )'" ; done
''
' foo-bar '
' foo_baz '
' foo1.2 '
''
```

SUSE never provided ReaR officially for SLE10
and "Version 1.16.1 (June 2014)" is the last version that mentiones SLES10
as somehow supported in the release notes and then
```
Rear-1.17 dropped officially support for
...
SLES 9 and 10
```
see
https://github.com/rear/rear.github.com/blob/master/documentation/release-notes-1-17.md

Summary:
Using `\s` in grep search strings is OK.

Reply:
@jsmeix thank you for checking, but earlier you wrote that your usual method is to add spaces to the pattern in similar cases. Shouldn't the same be used here for consistency?

Reply:
I perfer simple spaces when grepping in disklayout.conf
because ReaR creates it with spaces as field seperators
and I assume users won't use e.g. tabs as field separartors
when they modify disklayout.conf.

But technically `\s` is better because
```
# echo -e "\tpace\t" | grep "\space\s" && echo Y || echo N
        pace
Y

# echo -e "\tpace\t" | grep " pace " && echo Y || echo N
N
```
so the code from @rmetrich is better than what I usually do.

On the other hand the code with simple spaces is easier to read
cf. my play with words above 'pace' vs. 'space'.